scripts/gulpfile.factory.js

```diff
@@ -271,18 +244,24 @@ ${fs
   .join("")}*/\n\n`
         )
       )
-      .pipe(dest(paths.compiled));
+      .pipe(dest(paths.temp));
   });
 
-  task("copy", () => {
-    return src(["*.js", `!${elementName}.js`], {
+  task("copy:src", () => {
+    return src(["*.js", `!${elementName}*.js`], {
       cwd: paths.source
-    }).pipe(dest(paths.compiled));
+    }).pipe(gulpif((file) => files.length > 0 && gulpmatch(file, files) || files.length === 0, dest(paths.compiled), dest(paths.temp)));
```

Comment:
`pfe-avatar` is not building because of this line. Why wouldn't we want the other js files to also go to the temp directory?

Reply:
@castastrophe Can you help me understand what this ternary is doing?